app/views/projects/index.html.erb

```diff
@@ -1,3 +1,7 @@
+<% @projects.each do |project| %>
+  <%= turbo_stream_from project %>
+<% end %>
```

Comment:
@jorgemanrubia When I open the projects index (http://localhost:3000/projects) in two separate browser windows, and in one of them I create a new project, the other one does not update with the new project. I have to manually refresh the page to see it.

I assume it's because of this block here ^, it only subscribes to the specific projects which exist at the time the page loads so it's not subscribed to a newly created project. One solution would be to introduce a new higher-level abstraction called Portfolio (a collection of projects) and broadcast refreshes from that. But that seems heavy-handed if we don't otherwise need a new model.

Is there a simpler solution to this within the Turbo 8 Morphing world?

Reply:
Yes. Check this PR https://github.com/hotwired/turbo-rails/pull/521. You can use the pluralized model name as the stream name for listening to additions.

Reply:
But would this also work when we have scopes in place? For example like:
```ruby
  <%= turbo_stream_from @projects.not_deleted.is_active.where(client_id: 1) %>
```
Or would it just subscribe to all the projects?